pkg/console/operator/operator.go

```diff
@@ -68,7 +68,8 @@ type consoleOperator struct {
 	infrastructureClient configclientv1.InfrastructureInterface
 	versionGetter        status.VersionGetter
 	// recorder
-	recorder events.Recorder
+	recorder       events.Recorder
+	resourceSyncer resourcesynccontroller.ResourceSyncer
```

Comment:
It's unclear to me why we need to sync the config map. I'd think we would only need to do that if we were mounting it into a pod. In our case, we can read it in place in the operator.